src/shared/utilities/editorUtilities.ts

```diff
@@ -15,3 +16,41 @@ export function getTabSizeSetting(): number {
 export function getInlineSuggestEnabled(): boolean {
     return Settings.instance.getSection('editor').get('inlineSuggest.enabled', true)
 }
+
+/**
+ * Unlike vscode.window.visibleTextEditors, which will only return the "visible" files in the IDE, this function will return all files opened in the IDE.
```

Comment:
Why doesn't `vscode.workspace.textDocuments` work for this use-case?

Reply:
let me verify it, I remember it doesn't work but kinda forget hot it failed

Reply:
@justinmk3 , `vscode.workspace.textDocuments` kinda return similar result to `vscode.workspace.visibleTextEditors`
![image](https://github.com/aws/aws-toolkit-vscode/assets/96078566/13d95769-19cd-4714-a51a-08df4ad3c3ae)

Reply:
in summary, either `vscode.workspace.textDocuments` or `vscode.worksapce.visibleTextEditors` will not return all opened files like we have with tabGroups API